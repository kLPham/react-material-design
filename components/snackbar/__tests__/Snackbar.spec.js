import React from 'react';
import { shallow } from 'enzyme';
import Snackbar from '../index';

describe('<Snackbar />', () => {
    it('should render with a message', () => {
        const message = 'this is my message';
        const snackbar = shallow(
            <Snackbar message={message} />,
    );
        expect(snackbar).toMatchSnapshot();
    });
    it('should render with actionText', () => {
        const message = 'this is my message';
        const actionText = 'submit';
        const snackbar = shallow(
            <Snackbar message={message} actionText={actionText} />,
    );
        expect(snackbar).toMatchSnapshot();
    });
});
