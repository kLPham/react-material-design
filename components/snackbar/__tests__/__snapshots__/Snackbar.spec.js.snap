// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Snackbar /> should render with a message 1`] = `
<div
  aria-atomic="true"
  aria-hidden={true}
  aria-live="assertive"
  className="mdc-snackbar"
  onClick={[Function]}
>
  <div
    className="mdc-snackbar__text"
  >
    this is my message
  </div>
</div>
`;

exports[`<Snackbar /> should render with actionText 1`] = `
<div
  aria-atomic="true"
  aria-hidden={true}
  aria-live="assertive"
  className="mdc-snackbar"
  onClick={[Function]}
>
  <div
    className="mdc-snackbar__text"
  >
    this is my message
  </div>
  <div
    aria-hidden={true}
    className="mdc-snackbar__action-wrapper"
  >
    <button
      className="mdc-button mdc-snackbar__action-button"
      type="button"
    >
      submit
    </button>
  </div>
</div>
`;
